eo2js/src/commands/link.js

```diff
@@ -52,22 +52,33 @@ const pckg = function(options) {
 const link = function(options) {
   options = {...program.opts(), ...options}
   const project = path.resolve(options.target, options.project)
+  console.log(`Building npm project in: ${project}`)
+
+  console.log('Creating package.json...')
```

Comment:
@HaidarJbeily7 let's move this log to 3 lines below and print something like: `created package.json in ...` where ... is path